unified-codes/apps/web/src/components/molecules/Footer.tsx

```diff
@@ -43,10 +55,9 @@ const getStyles = (theme: Theme) => ({
 
 export type Footer = React.FunctionComponent<FooterProps>;
 
-const StyledDrawer = withStyles(getStyles)(Drawer);
 const _Footer: Footer = ({ classes }) => {
   return (
-    <StyledDrawer anchor="bottom" variant="permanent">
+    <Paper className={classes.root} square elevation={0}>
```

Comment:
At this stage premature, but what do you think about `elevation={0}` etc. being something that is imported rather than specified inline? Will it be shared between `<Paper>` components or differ throughout the app?